### ISSUE TITLE
Save planned maintenance in separate file before deleting

### ISSUE THREAD

**Issue by johannaengland** [Uninett/zino#202]
In Zino1 there is a function that is not implemented that is supposed to dump the state of a planned maintenance to a file before deleting the planned maintenance (just like with events).

**Comment by runborg** [comment]
in this PM file, is it possible to get a list of the ids for tickets created and ignored by this PM?

**Comment by johannaengland** [comment]
> in this PM file, is it possible to get a list of the ids for tickets created and ignored by this PM?

Yes, since one of the attributes of a PM is `event_ids`, which is a list that contains all events created/ignored by this PM and that will be dumped as well.
